### PR TITLE
change getRandomEmail

### PR DESCRIPTION
**Pull request by gazaret**
Old random method return inccorect values:

Math.random().toString(36).substring(15) // ""  
Math.random().toString(36).substring(8) // "57slj" 

I changed the method that would return the correct value.

I add new optional arg - prefix, in case when you need a specific name:

getRandomEmail(['@example.com'], 10, 'organization') // "organization-lyql2bla1u@example.com"

### CODE REVIEW

**Comment by xomyaq** [comment]
Hi, Gazaret!

Really good job with this "organization" thing and fixing with relatively long random strings generator issue.
I'll be grateful, if you would fix linter warnings and send PR again.

File: index.js [hunk 1]
```diff
@@ -44,10 +44,22 @@ function getEmailHash(email) {
  * Generates random email in given domains
  * @param {Array} domains
  * @param {number} [len=7]
+ * @param {string} prefix
  * @returns {string}
  */
-function getRandomEmail(domains, len = 7) {
-  const name = Math.random().toString(36).substring(len);
+function getRandomEmail(domains, len, prefix) {
+  if(!len) { len = 7; }
```
Comment (xomyaq) [inline index.js]: There are parameters default value in Node 6+. Let's use ES6 power! 😄

File: index.js [hunk 2]
```diff
@@ -44,10 +44,22 @@ function getEmailHash(email) {
...
+  if(!prefix) { prefix = ''; }
+  
+  const alfabet = '1234567890abcdefghijklmnopqrstuvwxyz';
+  
+  let name = !prefix ? '' : prefix + '-';
+  
+  for(let i = 0; i < len; i++) {
+    let randomChar = Math.round(Math.random() * (alfabet.length - 1));
```
Comment (xomyaq) [inline index.js]: `randomChar` declared as `let` but never changed in block. `const` is more suitable in this case.

**Comment by xomyaq** [comment]
Oh! And add tests covering this case please! 😇

**Comment by Gazaret** [comment]
Done :) 
Oh, conflicts!
I don't see any conflicts on my fork :(

**Comment by i11v** [comment]
Thank you for great PR! 😉

**Pull request merged**